src/engine/Engine.cpp

```diff
@@ -973,35 +972,12 @@ bool Engine::calculateBounds( InputQuery &inputQuery )
     return true;
 }
 
-void Engine::informConstraintsOfInitialBounds( InputQuery &inputQuery ) const
```

Comment:
Moving this to Preprocessor. This is the first thing we do now in the Preprocessor.